glue_jupyter/bqplot/common/tools.py

```diff
@@ -19,12 +19,27 @@ def __init__(self, viewer):
         self.viewer = viewer
 
     def activate(self):
+
+        # Disable any active tool in other viewers
+        for viewer in self.viewer.session.application.viewers:
+            if viewer is not self.viewer:
+                viewer.toolbar.active_tool = None
+
         self.viewer._mouse_interact.next = self.interact
 
     def deactivate(self):
         self.viewer._mouse_interact.next = None
 
 
+class BqplotSelectionTool(InteractCheckableTool):
+
+    def activate(self):
+        # Jumps back to "create new" if that setting is active
+        if self.viewer.session.application.get_setting('new_subset_on_selection_tool_change'):
```

Comment:
I've now made this optional as I think we don't necessarily want that behavior here by default. It can be enabled by passing ``new_subset_on_selection_tool_change=True`` to ``JupyterApplication.__init__`` when used in jdaviz. I've reverted the changes to the tests as the default behavior is now unchanged.